analyzers/src/SonarAnalyzer.VisualBasic/Rules/InsteadOfAny.cs

```diff
@@ -56,4 +66,22 @@ public sealed class InsteadOfAny : InsteadOfAnyBase<SyntaxKind, InvocationExpres
 
     protected override SyntaxNode GetArgumentExpression(InvocationExpressionSyntax invocation, int index) =>
         invocation.ArgumentList.Arguments[index].GetExpression();
+
+    protected override bool IsEntityFramework(SyntaxNode node, SemanticModel model)
```

Comment:
can you extract this in the base class?

`TryGetOperands` is in the `SyntaxFacade`, so it should allow for that

you'd only need another abstract method `IsInvocationExpression` method to be implemented by each class

Reply:
Good idea.
You actually don't even need `IsInvocationExpression`, you can do it via the Facade, check the implementation.